### DISCUSSION TITLE
Modal Flyout Position

### DISCUSSION THREAD

**Discussion by vincentcordel** [livewire/flux#755] [Feature Requests] [answered]
It'd be nice to be able to decide where to open the modal flyout. Right now, it can only be opened on the right. 
Add a variant for left, right, top, bottom ?

**Comment by indigoram89** [comment]
+1

**Comment by calebporzio** [comment, marked as answer]
Good call, this will be in the next release (only supporting left, right, and bottom now though)

![CleanShot 2024-11-30 at 22 18 57@2x](https://github.com/user-attachments/assets/730a5b5e-acae-44bb-af09-d7f6b3ad11af)

**Comment by jradtilbrook** [comment]
Woo this is awesome, thank you :raised_hands: 

Is anyone willing to provide some advice on how to conditionally set `right` or `bottom` based on viewport? I'm assuming/hoping some sort of special targeting of the `[data-flux-flyout]` selector as opposed to publishing the component to modify it
